.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## PR Description: *short_description*
+
+### Tickets:
+ - *ticket-id*
+
+### Details:
```

Comment:
I would move this section to the very begging. It gives information about the PR purpose together with PR title.

Reply:
Good catch